apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -75,16 +75,30 @@
                     if (usernameUserInput) {
                         var usernameUserInputValue = usernameUserInput.value.trim();
 
-                        if (!getParameterByName("isSaaSApp") && tenantName) {
+                        if (getParameterByName("isSaaSApp") && (getParameterByName("isSaaSApp") === "false")
```

Comment:
Is the value always going to be of type string? If unsure, we can cast the value to a string and to lower case before doing the check.

```js
String(getParameterByName("isSaaSApp")).toLowerCase()  === "false"
```